### DISCUSSION TITLE
AutoDJ is playing previous song when streamer disconnects / Refresh Deelays

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#6760] [General]
Helllo!

Anybody else noticed this behaviour in the latest version? When a streamer disconnects, and the AutoDJ starts to play, part of the latest song (the part where AutoDJ previously stopped), is still playing for a few seconds, mixed with the jingle or song that should normaly start. 

Also, i noticed some refresh/update deelays for about 30 seconds in Now Playing, Listeners Reports, Upcoming Song Queue. This reminds me of an old problem in the older versions, when now playing updates were too slow.

Hope you could identify this behaviour and fix it somehow. 

Many thanks!

**Comment by RadioHerzblut** [comment]
I have the same thing with mixing the song after the handover.

**Comment by unudelaradio** [comment]
What i also noticed is that when a streamer reconnects, the AutoDJ isn't fade out anymore... for that smooth transition.

**Comment by netbladenl** [comment]
I've created an issue for this ; https://github.com/AzuraCast/AzuraCast/issues/6763

**Comment by Moonbase59** [comment]
Possibly related? → https://github.com/AzuraCast/AzuraCast/issues/6565

**Reply by netbladenl** [reply]
Looks a lot like it, I'm not use if it is a Liqiudsoap error or an Azuracast error/

**Comment by phonic2k** [comment]
I also have the same annoying issue.  If you have live streamers, then It's best to "Disable Crossfading" in AutoDJ until crossfading is fixed for live streams.
